universal-login-wallet/src/core/services/StorageService.ts

```diff
@@ -0,0 +1,35 @@
+import {cast, Sanitizer} from '@restless/sanitizers';
+
+export class StorageService {
+  get(key: string): string | null {
+    return localStorage.getItem(key);
+  }
+
+  set(key: string, value: string) {
+    localStorage.setItem(key, value);
+  }
+}
+
+export class StorageEntry<T> {
```

Comment:
Abstraction level 10000 👍 😄

Reply:
`StorageEntry` is just a nice interface that bundles up together a storage key and a sanitizer for a specific value to prevent mixups and type-unsafe value casts.